public/pages/Dashboard/Container/DashboardOverview.tsx

```diff
@@ -175,6 +174,13 @@ export function DashboardOverview() {
     intializeDetectors();
   }, []);
 
+  useEffect(() => {
+    if (errorGettingDetectors) {
+      toastNotifications.addDanger('Unable to get all detectors');
```

Comment:
can you log the `errorGettingDetectors ` here?

Reply:
Sure, will add.